.github/workflows/python_package.yml

```diff
@@ -156,6 +188,30 @@ jobs:
             artifact-name: 'windows-amd64-cli-and-liblightgbm'
             python_version: '3.11'
     steps:
+      - name: Install packages used by third-party actions
```

Comment:
Copied from the R-package jobs, after observing that `git` was too old for `actions/checkout` in the `ubuntu:22.04` image.

https://github.com/microsoft/LightGBM/blob/9124af3e61bc7deae4bf7538c8a872b11f4e3a2c/.github/workflows/r_package.yml#L140